personal/Adam/Reports/zajecia2.Rmd

```diff
@@ -0,0 +1,59 @@
+---
+title: "Zajecia2"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Zadanie 1
+
+Wyświetlcie mapę krajów Europejskich  wg liczby ludności
+
+
+
+```{r cars}
+library(sf)
+library(raster)
+library(spData)
+EU<- world[world$continent=="Europe","pop"]

```

Comment:
library() przenieś następnym razem do osobnego bloku niż przetwarzanie danych. Linijka 20 powinna mieć spacje wokół <-, = i po przecinku:
EU <- world[world$continent == "Europe", "pop"]

Reply:
To są dobre tzw. dobre praktyki pisania kodu, na które warto zwracać uwagę kiedy się pisze kod do pracy z innymi osobami.